apps/frontend/src/network/users/__mocks__/api.ts

```diff
@@ -17,3 +21,13 @@ export const patchUser = jest.fn(
     };
   },
 );
+
+export const postUserAvatar = jest.fn(
+  async (id: string, post: UserAvatarPostRequest): Promise<UserResponse> => {
+    const user = await getUser(id);
+    return {
+      ...user,
+      avatarUrl: `url: ${post.avatar}`,
```

Comment:
Assuming `post.avatar` is a URL already?
```suggestion
      avatarUrl: post.avatar,
```

Reply:
We submit a base64 string and squidex gives us a random url back. We can't exactly do that so ive returned whats sent with a prefix of url:

Reply:
Yeah that's true of course, it doesn't have to be the same (a data URL) but can be any URL. Just thought we'd make it a bit more realistic that way (could of course also be `https://example.com/someimage.jpg`). But I guess `url: data:base64;bla` is also technically a valid URL with protocol `url` so whatever :joy: